### DISCUSSION TITLE
group discussion error

### DISCUSSION THREAD

**Discussion by prototypeninja** [salman0ansari/whatsapp-api-nodejs#223] [Q&A]
Good evening.
I have a problem with the node js server since the last update. Before I used the version with Baileys 3.0.0 which stopped working for a while so I updated the server from the git repository. My problem is that I can't retrieve messages from group discussions. please help me please.
here is the error i am getting

 _chains: {
    'BSqeZeCl/syWAwqz2AqZ+EAaAC58zuAmBi6pRInz1cI2': { chainKey: [Object], chainType: 2, messageKeys: {} },
    'BSc4/vDt+YAZFev0DZLVE5aV4c6HNAa1SFAgOuXO4A94': { chainKey: [Object], chainType: 1, messageKeys: {} }
  },
  registrationId: 797090690,
  currentRatchet: {
    ephemeralKeyPair: {
      pubKey: <Buffer 05 27 38 fe f0 ed f9 80 19 15 eb f4 0d 92 d5 13 96 95 e1 ce 87 34 06 b5 48 50 20 3a e5 ce e0 0f 78>,
      privKey: <Buffer 38 87 3f 05 d6 e4 4b 49 f4 b5 f2 a9 bf 8c 1b 87 e4 fe 8d 50 ae 35 09 fd e6 f1 44 50 4d 2c a1 60>
    },
    lastRemoteEphemeralKey: <Buffer 05 2a 9e 65 e0 a5 fe cc 96 03 0a b3 d8 0a 99 f8 40 1a 00 2e 7c ce e0 26 06 2e a9 44 89 f3 d5 c2 36>,
    previousCounter: 0,
    rootKey: <Buffer aa 6a b9 e5 57 c7 1c f1 a7 83 84 c5 7a 76 ed 4f bd 2c fa 24 5c cb ce 34 66 92 4f 59 ac 2f 7f d3>
  },
  indexInfo: {
    baseKey: <Buffer 05 78 37 2b d1 99 1f 37 5c d3 8f 4e 3f 75 17 92 e1 a8 ba 51 7d 66 fb 38 fe ce 6c 42 69 e5 56 34 6e>,
    baseKeyType: 2,
    closed: -1,
    used: 1658844700777,
    created: 1658844700777,
    remoteIdentityKey: <Buffer 05 a6 a2 b5 1c 54 55 ca 21 51 21 ba ad f9 26 40 0d 2f b9 fb 39 9f 1b 77 c1 60 93 e6 de c7 81 81 36>
  }
}
Closing open session in favor of incoming prekey bundle
Closing session: SessionEntry {
  _chains: {
    BcfEpDzthWSX0SEuOEzbkfvH9hYagtL71UK2547Uh0Q4: { chainKey: [Object], chainType: 2, messageKeys: {} },
    'BX7h4fLy1B1PXKvHoAGHNPe91k5BLE/heIIBP3x4Uk8F': { chainKey: [Object], chainType: 1, messageKeys: {} }
  },
  registrationId: 797090690,
  currentRatchet: {
    ephemeralKeyPair: {
      pubKey: <Buffer 05 7e e1 e1 f2 f2 d4 1d 4f 5c ab c7 a0 01 87 34 f7 bd d6 4e 41 2c 4f e1 78 82 01 3f 7c 78 52 4f 05>,
      privKey: <Buffer 88 b9 33 01 59 69 0f af 58 bf 0d 59 62 c3 12 01 f0 5e fd 23 58 60 b1 f3 d5 83 d0 13 1e 21 db 43>
    },
    lastRemoteEphemeralKey: <Buffer 05 c7 c4 a4 3c ed 85 64 97 d1 21 2e 38 4c db 91 fb c7 f6 16 1a 82 d2 fb d5 42 b6 e7 8e d4 87 44 38>,
    previousCounter: 0,
    rootKey: <Buffer a8 39 36 2c e7 7f a6 5f 4f 41 6b 5d c2 5e b2 96 26 c4 5c dc 1c 59 14 93 41 32 4e 76 c9 57 a0 e5>
  },
  indexInfo: {
    baseKey: <Buffer 05 b0 74 32 f3 b6 6a 8d 45 d4 36 bf ce 39 d7 ef 04 e1 d8 84 88 78 1d 17 8c cb d9 05 bb b0 d6 dc 31>,
    baseKeyType: 2,
    closed: -1,
    used: 1658844702938,
    created: 1658844702938,
    remoteIdentityKey: <Buffer 05 a6 a2 b5 1c 54 55 ca 21 51 21 ba ad f9 26 40 0d 2f b9 fb 39 9f 1b 77 c1 60 93 e6 de c7 81 81 36>
  }
}

**Comment by salman0ansari** [comment]
this is not an error its part of log

**Reply by prototypeninja** [reply]
Okay. what should i do to get messages from groups

**Reply by salman0ansari** [reply]
webhook?

**Reply by prototypeninja** [reply]
I didn't understand

**Reply by salman0ansari** [reply]
please be more specific what exactly you want to do

**Reply by prototypeninja** [reply]
I would like to receive message content from groups and send them to a webhook. such as private chats

**Reply by prototypeninja** [reply]
Can somebody help me please?
